packages/theme/stories/foundations/size.stories.mdx

```diff
@@ -12,8 +12,10 @@ All size values are built on the basis of 4px, `--salt-size-basis-unit`. We enco
 
 <DocGrid>
   <SpacingBlock spacingVar="--salt-size-accent" />
+  <SpacingBlock spacingVar="--salt-size-adornment-gap" />
```

Comment:
adornmentGap wasn't documented, so I don't feel the need to document the deprecation

Reply:
I think the non-documentation was a bug. We should document it as deprecated :)